### ISSUE TITLE
Loss Function.md

### ISSUE THREAD

**Issue by zhpmatrix** [Py-Learning/python-computing#11]
**loss func = loss term + regualarization term**
common loss functions:

*1.* Gold Standard (ideal case)

*2.* Hinge (SVM, soft margin)未深入

*3.* Log (logistic regression, cross entropy error) 本质上是一致的。

*4.* Squared loss (linear regression)很常见

*5.* Exponential loss (Boosting)相比于理想类型，在误差项的下边做了自然对数为底的变化
